src/main/java/org/alfresco/rest/core/RestWrapper.java

```diff
@@ -1157,4 +1157,9 @@ public void configureAlfrescoEndpoint()
         this.serverPort = restProperties.envProperty().getPort();
         configureServerEndpoint();
     }
+
+    public boolean isResponseWithEmptyArray(Response response) {
+        return (response.jsonPath().get() instanceof List && ((List<?>)response.jsonPath().get()).isEmpty()) ||
+                (response.jsonPath().get() instanceof Map && ((Map<?,?>)response.jsonPath().get()).isEmpty());
```

Comment:
It would be useful to include some Javadoc for this method to explain that the `get` method can only return:
> The object matching the Object graph. This may be any primitive type, a List or a Map.

That is why it's sufficient to only check List and Map.